src/ontology/cl-edit.owl

```diff
@@ -3533,9 +3533,9 @@ SubClassOf(obo:CL_0000032 ObjectSomeValuesFrom(obo:RO_0002202 obo:CL_0000114))
 
 # Class: obo:CL_0000033 (apocrine cell)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:tfm"^^xsd:string) Annotation(oboInOwl:hasDbXref "ISBN:0721662544"^^xsd:string) obo:IAO_0000115 obo:CL_0000033 "An apocrine cell is a secretory cell characterized loss of part of the cytoplasm during the process of secretion - secretions are budded off through the plasma membrane producing membrane bound vesicles containing the secreted substance."^^xsd:string)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:tfm"^^xsd:string) Annotation(oboInOwl:hasDbXref "ISBN:0721662544"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:25960390"^^xsd:string) obo:IAO_0000115 obo:CL_0000033 "An apocrine cell is an exocrine cell characterized by loss of part of the cytoplasm during the process of secretion. The secreted substance is accumulated at the apical end and budded off through the plasma membrane producing secreted vesicles."^^xsd:string)
```

Comment:
Thanks for cleaning up this definition. 
Some feedback:

- Definition can start with "An exocrine cell characterized by... ".  Starting with "An apocrine cell is" is not required.
- The reference states: "According to various interpretations and views which developed over time, the secretory material may be contained in secretory vesicles **or dissolved in the cytoplasm that is lost during secretion** [[82]](https://www.sciencedirect.com/science/article/pii/S0304416515001282?via%3Dihub#bb0410), [[83]](https://www.sciencedirect.com/science/article/pii/S0304416515001282?via%3Dihub#bb0415), [[84]](https://www.sciencedirect.com/science/article/pii/S0304416515001282?via%3Dihub#bb0420). Upon secretion the materials are released as cytoplasmic fragments into the glandular lumen or interstitial space [[78]](https://www.sciencedirect.com/science/article/pii/S0304416515001282?via%3Dihub#bb0390)." 
I think this nuance could be incorporated in the second sentence of the definition to help differentiate this process from exocytosis.